src/run-parallel.js

```diff
@@ -1,14 +1,4 @@
-const logger = require('./logger');
-const Stopwatch = require('./stopwatch');
-
-const timer = new Stopwatch();
-
-module.exports = async (tasks = []) => {
-	logger.message(`Running ${tasks.length} tasks in parallel`);
-
-	timer.start();
-	await Promise.all(tasks.map(Reflect.apply));
-	timer.stop();
-
-	logger.message(`${tasks.length} tasks complete, took ${timer.duration}s`);
+module.exports = (tasks = []) => {
+	// TODO: support concurrency level (maybe async-sema?)
+	return Promise.all(tasks.map(Reflect.apply));
```

Comment:
TIL `Reflect.apply`. but, won't this call the tasks with the index as the context and the array of tasks as the arguments?

Reply:
That's right. It doesn't cause any issues right now as tasks are anon arrow functions with zero args, but I'll change it to avoid any potential future issues.